src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4313,11 +4313,11 @@ OBA:2045266	carotid artery wall area attribute	UBERON:0037094	wall of common car
 OBA:2045267	cerebral cortex area attribute	UBERON:0000956	cerebral cortex	PATO:0001323	area			https://orcid.org/0000-0001-8314-2140
 OBA:2045268	striated muscle fiber area attribute	CL:0000737	striated muscle cell	PATO:0001323	area			https://orcid.org/0000-0001-8314-2140
 OBA:2045269	aortic vascular smooth muscle cell calcification attribute	CL:0002539	aortic smooth muscle cell	PATO:0001447	calcified			https://orcid.org/0000-0001-8314-2140
-OBA:2045270	basal ganglia growth attribute	UBERON:0002420	basal ganglion	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045271	brain growth attribute	UBERON:0000955	brain	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045272	cerebellum growth attribute	UBERON:0002037	cerebellum	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045273	cerebral cortex growth attribute	UBERON:0000956	cerebral cortex	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
-OBA:2045274	prefrontal cortex growth attribute	UBERON:0000451	prefrontal cortex	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
+OBA:2045270	basal ganglia growth characteristic	UBERON:0002420	basal ganglion	PATO:0001492	growth rate			https://orcid.org/0000-0001-8314-2140
```

Comment:
@rays22 Why were 'attribute' terms replaced by 'characteristic'?

Reply:
It sound better. It was suggested by James at the meeting last Friday.